src/site/content/en/blog/optimize-long-tasks/index.md

```diff
@@ -294,6 +294,70 @@ async function saveSettings () {
 
 With this approach, you get a fallback for browsers that don't support `isInputPending()` by using a time-based approach that uses (and adjusts) a deadline so that work will be broken up where necessary, whether by yielding to user input, or by a certain point in time.
 
+### Frame-aware yielding
+
+Now for a more nuanced approach, which is something you could call "frame-aware" yielding. This yielding technique is a bit different than the ones outlined previously. Yet, the effects of it are more aware of when a frame needs to be presented, and does its best to yield when this is the case:
+
+```js
```

Comment:
Out of curiosity, why did you change this from the original [strawman](https://gist.github.com/mmocny/d54c381d46e69fb7d1337a4bd566f6c7) @mmocny proposed? Was this something you guys discussed in the meeting?

Reply:
FWIW I'm adding a hopefully better strawman (but still strawman) into my lightning talk from perf.now(), will likely publish today so we can look at it.